cli/src/lib/npm/npmLibDefs.js

```diff
@@ -329,12 +329,15 @@ async function _npmExists(pkgName: string): Promise<Function> {
   return got(pkgUrl, {method: 'HEAD'});
 }
 
+export const CACHE_REPO_EXPIRY = 1000 * 60;
+
 export async function findNpmLibDef(
   pkgName: string,
   pkgVersion: string,
   flowVersion: FlowVersion,
+  useCacheUtil: number = CACHE_REPO_EXPIRY,
```

Comment:
I think this is now third place where default `CACHE_REPO_EXPIRY` is set.

Other places being:
- https://github.com/flow-typed/flow-typed/pull/3305/files#diff-8fda159841136e265107671260f39d80R157
- https://github.com/flow-typed/flow-typed/blob/master/cli/src/lib/cacheRepoUtils.js#L79

would https://github.com/flow-typed/flow-typed/blob/master/cli/src/lib/cacheRepoUtils.js#L79 be enough?